### DISCUSSION TITLE
Docker-compose for opa

### DISCUSSION THREAD

**Discussion by lamoboos223** [open-policy-agent/community#344] [OPA and Rego]
in the docs the [docker-compose](https://www.openpolicyagent.org/docs/latest/http-api-authorization/) uses the image `openpolicyagent/demo-restful-api:0.3` produces the error `Error response from daemon: manifest for openpolicyagent/demo-restful-api:0.3 not found: manifest unknown: manifest unknown`. When i checked the docker hub i can see the following tags but no `0.3` tag.

![image](https://user-images.githubusercontent.com/35597031/219326968-637e6030-ee27-45b3-943c-2a39679dca6f.png)

So i suggest to update the docs to the image `openpolicyagent/demo-restful-api` to pull the latest image.

**Comment by anderseknert** [comment]
Good catch, @lamoboos223! Would you like to submit a fix to the docs? File is over [here](https://github.com/open-policy-agent/opa/blob/main/docs/content/http-api-authorization.md?plain=1#L93). You can probably edit it right in the GitHub UI.

**Reply by lamoboos223** [reply]
Yes I would sir, I will do it today!
@anderseknert
